resources/views/api/openinghours/month.blade.php

```diff
@@ -0,0 +1,75 @@
+@if($data[0]['openinghours'])
+    <?php $firstDay = reset($data[0]['openinghours'])->date; ?>
+    <?php  $lastDay = end($data[0]['openinghours'])->date; ?>
+    @inject('localeService', 'App\Services\LocaleService')
+    <?php
+    $weekdays = \App\Models\DayInfo::WEEKDAYS_SHORT;
+    for ($i = 0; $i < $localeService->getWeekStartDay(); $i++) {
+        $value = array_shift($weekdays);
+        $weekdays[] = $value;}
+
+    ?>
+    <div class="openinghours openinghours--calendar">
+        <div class="openinghours--header">
+            <a href="#" class="openinghours--prev">@lang('openinghourApi.PREVIOUS')</a>
+            <div class="openinghours--month">@lang('openinghourApi.'.$firstDay->format('F')) {{ $firstDay->format('Y') }}</div>
+            <a href="#" class="openinghours--next">@lang('openinghourApi.NEXT')</a>
+        </div>
+        <div class="openinghours--days">
+            @foreach($weekdays as $weekday)
+                <div class="openinghours--day openinghours--day--day-of-week">@lang('openinghourApi.'.$weekday)</div>
+            @endforeach
+
+            @for($i=0;$i<((7 - $firstDay->dayOfWeek - $localeService->getWeekStartDay()) % 7);$i++)
+                <div class="openinghours--day openinghours--day-disabled"></div>
+            @endfor
+
+            @foreach($data[0]['openinghours'] as $dayInfoObj)
```

Comment:
Use `@include` for common template output? (see https://laravel.com/docs/5.5/blade#including-sub-views).